pennylane/templates/embeddings/amplitude.py

```diff
@@ -38,11 +36,6 @@ class AmplitudeEmbedding(StatePrep):
 
         On some devices, ``AmplitudeEmbedding`` must be the first operation of a quantum circuit.
 
-    .. warning::
```

Comment:
This is not valid anymore?

Reply:
Now it works the same as StatePrep and it is differentiable (although I tested AmplitudeEmbedding before the change  and also seemed to be differentiable. Maybe it was fixed and the warning was not deleted).